### ISSUE TITLE
Schaefer atlas MNI space

### ISSUE THREAD

**Issue by dominikkraft** [PennBBL/xcpEngine#381]
**Describe the bug**

This is prob. not a bug, but I was wondering about this recently & I´d great to get feedback and your ideas. 

As far as I understood correctly, Schaefer atlases are directly copied from [here](https://github.com/ThomasYeoLab/CBIG/tree/master/stable_projects/brain_parcellation/Schaefer2018_LocalGlobal) being in FSL MNI space. 

When running the fcon module one gets the **prefix_{atlas_name}_ts.1D** file for each atlas. 

Figuring out how this is done by looking up the logs, this code is used before to get the atlas from MNI space to native T1w space.

`antsApplyTransforms -e 3 -d 3 -i /xcpEngine/atlas/schaefer100x7/schaefer100x7MNI.nii.gz -r /path/sub-aam063_struct.nii.gz -o /path/sub-aam063_atlas/sub-aam063_schaefer100x7.nii.gz -n MultiLabel -t /path/sub-aam063_from-MNI152NLin2009cAsym_to-T1w_mode-image_xfm.h5 `

Following the question on [neurostars](https://neurostars.org/t/parcellation-for-mni152nlin2009casym/519/7) and readings from [here](https://www.lead-dbs.org/about-the-mni-spaces/) FSL MNI and fmripreps MNI 2009c space are not identical. According to the code above, the **.h5** file from fmriprep is used to transform Schaefer100x7MNI_FSL to native space, potentially introducing (subtle) accuracies errors as both underlying templates are different. 

Given that this is correct (please let me know if not!) wouldn´t it be better to use the Schaefer templates build from fmripreps mni152nlin2009casym template? They should be available on [templateflow](https://github.com/templateflow/templateflow) now. 

I think one could also use `roiquant` with a customized atlas (i.e., one from templateflow) or some other routine, but I really liked the functionality (as I was only about to denoise my data with the engine) and maybe this would be the 'cleaner' way.

Best,
Dominik

**Comment by a3sha2** [comment]
Hi @dominikkraft 

Thank you very much.  FSL MNI and fmripreps MNI 2009c  are not really identical, they just cover each other( when you view it in fsleyes). The MNI template use in  the xcp is the same as in the one in templateflow (MNI152NLin2009cAsym). We will  replace all the atlas in FSL MNI to -MNI152NLin2009cAsym.

**Comment by dominikkraft** [comment]
@a3sha2 thanks a lot! Just to get it right, the Schaefer atlas will be available in MNI152NLin2009cAsym upon the next release? When do u plan to upload it?

**Comment by a3sha2** [comment]
This is very important, it will be available before Wednesday!

**Comment by a3sha2** [comment]
can you pull for new docker or singularity image `pennbbl/xcpengine:latest`, the atlases have been replaced (now in MNI2009c space) 
https://github.com/PennBBL/xcpEngine/tree/master/atlas/schaefer100x7

**Comment by dominikkraft** [comment]
thank you very much! Appreciate your quick help! Maybe it´s worth mentioning this somewhere in the reference.bib or README or within the docs for future users?

**Comment by a3sha2** [comment]
Yes, we will mention it in the next release, it is very important
